### ISSUE TITLE
editor remembers tattoos of last edit

### ISSUE THREAD

**Issue by DarinBeard** [pedr0fontoura/fivem-appearance#78]
if I edit a character and add tattoos or edit one with tattoos and save it. If I load another character into the editor, even if I make sure the tattoos data has no tattoos in it, it still remembers the last tattoo selections and applies them to the new character when saved. How can I clear what was there before so it doesn't propagate to a new character?

**Comment by franfdezmorales** [comment]
You are right, currently the system is designed so that you can only use 1 ped per user, in case you change character, the tattoos should change, I will take a look at this.

**Comment by Mrwimz** [comment]
Any updates of this, i am using a fork for better esx compatability, wasabis version, so if the issue lies in that i understand if you can't help me.
